tests/utils.py

```diff
@@ -383,7 +385,7 @@ def train_step(images, labels):
 
 
 def get_image_classifier_kr(
-    loss_name="categorical_crossentropy", loss_type="function_losses", from_logits=False, load_init=True
+        loss_name="categorical_crossentropy", loss_type="function_losses", from_logits=False, load_init=True
```

Comment:
Please don't add indent, this is following Black style.
```suggestion
    loss_name="categorical_crossentropy", loss_type="function_losses", from_logits=False, load_init=True
```